### PR TITLE
[doc] Release notes for 0.1.9

### PR DESCRIPTION
**Pull request by ejgallego**

### CODE REVIEW

File: etc/release_notes/0.1.9.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
+Dear all,
+
+we are happy to announce the 0.1.9 release of `coq-lsp`.
```
Comment (Alizter) [inline etc/release_notes/0.1.9.md:3]:
```suggestion
We are happy to announce the 0.1.9 release of `coq-lsp`.
```

File: etc/release_notes/0.1.9.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+
+This release brings many new features and fixes, in particular:
+
+- new on-demand checking mode: `coq-lsp` can now check files on
```
Comment (Alizter) [inline etc/release_notes/0.1.9.md:7]:
```suggestion
- New on-demand checking mode: `coq-lsp` can now check files on
```

File: etc/release_notes/0.1.9.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+  demand, either by following the goals requested, or by following the
+  current viewport of your editor. Combined with the new keybinding
+  `M-n/M-p` for moving between Coq sentences, this provides a mode
+  similar to the usual one in Proof General. Additionally, we now show
+  real-time server status and checking information in the VSCode
+  language status area.
+
+- new interruption support using `memprofs-limits` (only in OCaml
```
Comment (Alizter) [inline etc/release_notes/0.1.9.md:15]:
```suggestion
- New interruption support using `memprofs-limits` (only in OCaml
```

File: etc/release_notes/0.1.9.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  Arias, Guillaume Baudart, and Laetitia Teodorescu; thanks to Alex
+  Sanchez-Stern).
+
+- Coq meta commands `Reset` / `Reset Inital` and `Back` are supported,
+  together with the incremental checking engine they do provide some
+  interesting document splitting and isolation features.
+
+- New user manual with some information on how to start with `coq-lsp`
+
+- improved support for VSCode Live Share; full support requires
```
Comment (Alizter) [inline etc/release_notes/0.1.9.md:33]:
```suggestion
- Improved support for VSCode Live Share; full support requires
```

File: etc/release_notes/0.1.9.md [hunk 5]
```diff
@@ -0,0 +1,67 @@
...
+  approval from Microsoft, please see below if you are interested in
+  helping with this.
+
+- new `Free Memory` server command.
```
Comment (Alizter) [inline etc/release_notes/0.1.9.md:37]:
```suggestion
- New `Free Memory` server command.
```

File: etc/release_notes/0.1.9.md [hunk 6]
```diff
@@ -0,0 +1,67 @@
...
+
+- server settings are now updated on the fly when edited in VSCode.
```
Comment (Alizter) [inline etc/release_notes/0.1.9.md:39]:
```suggestion
- Server settings are now updated on the fly when edited in VSCode.
```

File: etc/release_notes/0.1.9.md [hunk 7]
```diff
@@ -0,0 +1,67 @@
...
+
+- new heatmap feature to detect execution time hotspots in your Coq
```
Comment (Alizter) [inline etc/release_notes/0.1.9.md:41]:
```suggestion
- New heatmap feature to detect execution time hotspots in your Coq
```

File: etc/release_notes/0.1.9.md [hunk 8]
```diff
@@ -0,0 +1,67 @@
...
+  documents, by Ali Caglayan; plus many more improvements and fixes
+  w.r.t. performance monitoring.
+
+- locations are now stored in the server in protocol format, this
+  should solve some Unicode issues present in previous versions
+  (by E. J. Gallego Arias and Leo Stefanesco).
+
+- many improvements to both client and server plugin API
```
Comment (Alizter) [inline etc/release_notes/0.1.9.md:49]:
```suggestion
- Many improvements to both client and server plugin API
```

File: etc/release_notes/0.1.9.md (hunk 1 above)
Comment (ejgallego) [reply etc/release_notes/0.1.9.md:3]: Thanks a lot @Alizter , all the comments should be incorporated.

**Pull request merged**